index.php

```diff
@@ -66,6 +69,9 @@
  echo '<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />';
  echo '<meta http-equiv="refresh" content="305" />';
  echo '<meta property="og:site_name" content="'.$this_site.'" />';
+ echo '<meta name="twitter:label1" value="Reading time"><meta name="twitter:data1" value="1 min read">';
+ #echo '<meta name="twitter:label2" value="Updated"><meta name="twitter:data2" value="'.$theDate.'">';
+ echo '<meta property="article:published_time" content="2017-02-16T16:03:52+00:00" />';
```

Comment:
Thanks, and I will work on a dev sandbox site for changes like this from this point forward.